support-frontend/assets/helpers/forms/checkouts.ts

```diff
@@ -161,7 +161,7 @@ function getPaymentMethods(
 			return !isSupporterPlus ? [Stripe, PayPal, AmazonPay] : [Stripe, PayPal];
 		}
 
-		return [Stripe, PayPal];
+		return [Stripe, PayPal, AmazonPay];
```

Comment:
```
		if (
			contributionType === 'ONE_OFF' ||
			getQueryParameter('amazon-pay-recurring') === 'true'
		) {
			return !isSupporterPlus ? [Stripe, PayPal, AmazonPay] : [Stripe, PayPal];
		}

		return [Stripe, PayPal, AmazonPay];
```

I think something's got a bit lost in the conditions here- that if block is essentially meaningless. Under what circumstances specifically do we display AmazonPay?

Reply:
Originally this was to disable AmazonPay in the variant-only no matter what the RRCP switches settings were set to. Still need to replace the use of checkoutFormHasBeenSubmitted so I it left there, however, it probably should of been removed. Done it now, will test RRCP switch for AmazonPay affects control and variant in CODE